meson.build

```diff
@@ -45,9 +45,15 @@ conf.set_quoted('PACKAGE_NAME', meson.project_name())
 conf.set('PACKAGE_MAJOR', major_version)
 conf.set('PACKAGE_MINOR', minor_version)
 
-libcrypto = dependency('libcrypto', version: '>= 3.0.7')
+libcrypto = dependency('libcrypto', version: '>= 3.0.7', required: false)
+if not libcrypto.found()
+  libcrypto = dependency('libcrypto3', version: '>= 3.0.7', required: true)
+endif
 provider_path = libcrypto.get_variable(pkgconfig: 'modulesdir')
-libssl = dependency('libssl', version: '>= 3.0.7')
+libssl = dependency('libssl', version: '>= 3.0.7', required: false)
+if not libssl.found()
+  libssl = dependency('libssl3', version: '>= 3.0.7', required: true)
```

Comment:
Might it make sense to specify `method: 'pkg-config'`, as the "libssl3" system library name is taken by NSS? Afaik the Meson OpenSSL [detection](https://mesonbuild.com/Release-notes-for-0-62-0.html#new-custom-dependency-for-openssl) falls back to the system naming if pkg-config doesn't work.